test/powershell/Modules/Microsoft.PowerShell.Utility/Sort-Object.Tests.ps1

```diff
@@ -296,6 +296,18 @@ Describe 'Sort-Object Stable Unit Tests' -Tags 'CI' {
 			$results[1]  | Should -Be 1
 			$results[2]  | Should -Be 3
 		}
+		
+		It "Sort-Object with Uniqe Stable should return proper records"{
```

Comment:
```suggestion
		It "Sort-Object with Unique Stable should return proper records"{
```

Reply:
Thanks, I have fixed the spelling error and code style violations in 7e56f03.